content/en/api/service_level_objectives/service_level_objectives_create.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Create a service level objective
+type: apicontent
+order: 29.01
+external_redirect: /api/#create-a-service-level-objective
+---
+
+## Create a service level objective
+
+If you manage and deploy SLOs programmatically, it's easier to define the SLO in the Datadog UI and [export its valid JSON][1].
```

Comment:
@platinummonkey - I am not seeing anything on exporting JSON in your original docs (this section never existed that I can find), and I also can't find it in the UI to add it in. Is this functionality available yet?